ray_on_golem/provider/node_provider.py

```diff
@@ -59,6 +60,7 @@ def bootstrap_config(cls, cluster_config: Dict[str, Any]) -> Dict[str, Any]:
         ray_on_golem_client = cls._get_ray_on_golem_client_instance(
             provider_parameters["webserver_port"],
             provider_parameters["enable_registry_stats"],
+            provider_parameters.get("log_level", "info"),
```

Comment:
```suggestion
            provider_parameters["log_level"],
```
Parameter default value should be set next to the `webserver_port` block via `setdefault`.